src/components/common/StyledButton/index.js

```diff
@@ -19,14 +20,14 @@ const StyledButton = ({ disabled, onClick, type, iconClass, btnText }) => {
 
   return (
     <Button className={clsx(classes.styledButton, classes[buttonColor[type]])} disabled={disabled} onClick={onClick}>
-      <Icon className={iconClass} />
+      {iconClass ? <Icon className={iconClass} /> : null}
```

Comment:
Can this be handled within Icon class?

Reply:
Nope we handled that way.. but it is taking a position in the DOM and causing problem in the styling of the button.

Reply:
[raised issue](https://github.com/singnet/refactored-dapp/issues/38)